### PR TITLE
improve script creation and fix linux

### PR DESCRIPTION
**Pull request by carlodri**
Checklist
* [x] Used a fork of the feedstock to propose changes
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

supersedes #4 (maybe 🤔)

### CODE REVIEW

**Comment by carlodri** [comment]
@conda-forge-admin, please rerender

File: recipe/build.sh [hunk 1]
```diff
@@ -1,4 +1,5 @@
 #!/bin/sh
 mkdir $PREFIX/lib
-cp $RECIPE_DIR/bfg $PREFIX/bin
+echo "#!/bin/sh" >> $PREFIX/bin/bfg
+echo "java -jar ${PREFIX}/lib/bfg.jar \"$@\"" >> $PREFIX/bin/bfg
```
Comment (lebedov) [inline recipe/build.sh:6]: This should be changed to

```bash
echo "java -jar ${CONDA_PREFIX}/lib/bfg.jar \"$@\"" >> $PREFIX/bin/bfg
```

**Review by lebedov — CHANGES_REQUESTED** [review]
See my comment on `recipe/build.sh`

File: recipe/build.sh (hunk 1 above)
Comment (carlodri) [reply recipe/build.sh:6]: @conda-forge/core sorry for bothering you, but I'm not sure about the difference between `PREFIX` and `CONDA_PREFIX`, and consequently what to use in this case. It seems from the [docs](https://docs.conda.io/projects/conda-build/en/latest/user-guide/environment-variables.html#environment-variables-that-affect-the-build-process) that the latter is only relevant for the _build process_, and therefore, in this scenario, we should be using `PREFIX`. Can anyone give us some more insight?

Comment (scopatz) [reply recipe/build.sh:6]: Yeah, that's right, basically only use PREFIX, unless you are messing with conda itself

Comment (carlodri) [reply recipe/build.sh:6]: Thanks @scopatz!

**Comment by carlodri** [comment]
@lebedov let me know if this fixes linux builds

**Pull request merged**